index.src.html

```diff
@@ -382,8 +397,8 @@ <h4 id="examples-federated-signin">Federated Sign-in</h4>
 
     <h4 id="examples-post-signin">Post-sign-in Confirmation</h4>
 
-    To ensure that credentials are persisted correctly, it may be useful for a
-    website to tell the credential manager that sign-in succeeded.
+    To offer the user that credentials are persisted after a successful
```

Comment:
Perhaps: "To ensure that users are offered the opportunity to store new credentials after successful sign-in..."?